src/bp-messages/screens/inbox.php

```diff
@@ -24,6 +24,12 @@ function messages_screen_inbox() {
 		while ( bp_message_threads() ) :
 			bp_message_thread();
 			$thread_id = bp_get_message_thread_id();
+			$first_message             = BP_Messages_Thread::get_first_message( $thread_id );
```

Comment:
@meetmakadiya Instead of skipping the thread here can't we remove it directly in the query. Because by this way might be an issue with pagination.

Reply:
@dipskakadiya I didn't get your point can you explain more regarding the pagnation